package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "lcdgame",
+  "version": "0.3.4",
+  "description": "LCD game library in html5/javascript, to recreate simulations of 80s Game & Watch by Nintendo and other electronic handheld games like Tandy, Sunwing, Mini Arcade etc.",
+  "main": "bin/lcdgame.js",
```

Comment:
If you publish this repo as a module, you can use the bin directory as a publish source - so the module would contain the binary and it could be removed from git.